mlflow/store/tracking/abstract_store.py

```diff
@@ -490,6 +490,24 @@ async def log_spans_async(self, experiment_id: str, spans: list[Span]) -> list[S
         """
         raise NotImplementedError
 
+    def load_spans(self, trace_id: str) -> list[Span]:
+        """
+        Load spans for a given trace ID.
+
+        Returns:
+            List of Span entities.
+        """
+        raise NotImplementedError
+
+    async def load_spans_async(self, trace_id: str) -> list[Span]:
+        """
+        Asynchronously load spans for a given trace ID.
+
+        Returns:
+            List of Span entities.
+        """
+        raise NotImplementedError
```

Comment:
Do we need the async version of this API right now? It's unused. I added async for log spans because it's used by the MLflow FastAPI server

Reply:
I see, yea we don't need it right now I can remove it